tools/upgrade-mistral-wf-0.8

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+#
+## Mistral YAQL Migration tool
+#
+# This tool inspects a given Mistral workflow, parses it using a few
+# regex filters, and attempts to rewrite the file to adhere to the
+# updated YAQL syntax introduced in https://review.openstack.org/#/c/157741/
+#
+# There appear to be two different ways that the old style YAQL is
+# represented in workflows. This includes:
+#   * bare         - (node: $.node)
+#   * interpolated - (node: 'Node: {$.node}')
+#   * functions    - (node: range(0, $.nodes).list() )
+#
+# This command runs sed with two passes attempting to clean up a given
+# workflow. This command will also ensure a backup exists in the event
+# that things do not go as expected to prevent data loss.
+#######################################################################
+
+function usage {
+  echo "Usage: `basename $0` <workflow file>"
```

Comment:
Technically it's 'mistral file', workflows, workbooks, actions...
say workflow/workbook file?